packages/api/src/hooks/useAvailableMT5Accounts.ts

```diff
@@ -1,6 +1,12 @@
 import { useMemo } from 'react';
 import useFetch from '../useFetch';
 
+const market_type_to_leverage_mapper: Record<string, number> = {
+    gaming: 500,
+    financial: 1000,
+    all: 100,
+} as const;
```

Comment:
@thisyahlen-deriv The object has type, `as const` doesn't do anything here.
```suggestion
};
```